src/web_app/js/call.js

```diff
@@ -419,23 +419,45 @@ Call.prototype.maybeCreatePcClient_ = function() {
   if (this.pcClient_) {
```

Comment:
This function should probably be called maybeCreatePcClientAsync or some such to differentiate it from the synchronous createPcClient.